tests/plots/test_metric_plots.py

```diff
@@ -0,0 +1,56 @@
+import matplotlib
+import numpy as np
+import os
+
+from xplique.plots.metrics import barplot, fidelity_curves
+
+
+def test_bar_plot():
+    # test metrics barplot
+    cmap = matplotlib.cm.get_cmap("Set3")
+    methods_colors = {"method_" + str(i): cmap(i / 9) for i in range(10)}
+
+    scores = {}
+    for metric in ["metric_1", "metric_2"]:
+        scores[metric] = {
+            "method_" + str(i): np.random.random_sample()
+            for i in range(10)
+        }
+
+    filepath = "tests/plots/barplot_test.png"
+    barplot(
+        scores,
+        sort_metric="metric_1",
+        ascending=True,
+        title="test bar-plot",
+        filepath=filepath,
+        methods_colors=methods_colors
+    )
+
+    assert os.path.exists(filepath)
```

Comment:
This pass the CI ? If it does, super cool ;)

Reply:
What do yu call CI ? If it is tox, then yes it is successful